registry.json

```diff
@@ -4043,5 +4043,19 @@
    "telegram": "HappyRaspberry",
    "operatorName": "Patrik",
    "github": "kolesarp1"
-  }
+  },
+    {
+   "ticker": "PFL01",
+   "poolId": "24bfb62716d755bbf035fcb3f7e87a8d325fde0f04062e692aff3216",
+   "social": {
+     "website": "https://poolforlovelace.me/",
+     "twitter": "",
```

Comment:
Please can you add some sort of social media presence? This is one of the requirements we have for joining the alliance